azurelinuxagent/common/protocol/hostplugin.py

```diff
@@ -43,6 +43,8 @@
 HEADER_HOST_CONFIG_NAME = "x-ms-host-config-name"
 HEADER_ARTIFACT_LOCATION = "x-ms-artifact-location"
 HEADER_ARTIFACT_MANIFEST_LOCATION = "x-ms-artifact-manifest-location"
+HEADER_VERIFY_FROM_ARTIFACTS_BLOB = "x-ms-verify-from-artifacts-blob"
```

Comment:
Could we add  comments explaining what this header controls? I know the current headers don't have documentation, but let's do for the new ones,

Reply:
Added documentation